website/docs/docs/get-started/dbt-cloud-features.md

```diff
@@ -1,46 +1,110 @@
 ---
-title: "dbt Cloud features"
+title: "dbt Cloud benefits"
```

Comment:
What is motivation behind this change? "Benefits" can mean different things to different people and can often be a matter of opinion. Features might be better for instructional docs.

Reply:
great question! it came up during my coaching session with Ly so she suggested benefits. I'm happy with features too, might be less effort as well.